tests/fixtures/issues/issue-492.php

```diff
@@ -5103,7 +5103,7 @@
             'MinorVer' => '0',
             'Platform' => 'FirefoxOS',
             'Platform_Version' => 'unknown',
-            'Platform_Description' => 'Android OS',
+            'Platform_Description' => 'FirefoxOS',
```

Comment:
This one was sort of unexpected. When I added the platform description to FirefoxOS I expected all of the "unknown"s to need replacement, but not this one.

It must be some sort of inheritance related issue.